.github/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changes from release 2024/05 to 2024/06
 1. AIRAC (2406) - Update Fleetlands (EFVF) frequency - thanks to @rishab-alt
 2. AIRAC (2406) - Added Airbus Hangar and Helipads to Oxford (EGTK) SMR - thanks to @RedstonePilot (Ben Walker)
+3. AIRAC (2406) - Update Jersey (JSY) VOR coordinates - thanks to @chssn
```

Comment:
```suggestion
# Changes from release 2024/06 to 2024/07
1. AIRAC (2407) - Update Jersey (JSY) VOR coordinates - thanks to @chssn

# Changes from release 2024/05 to 2024/06
1. AIRAC (2406) - Update Fleetlands (EFVF) frequency - thanks to @rishab-alt
2. AIRAC (2406) - Added Airbus Hangar and Helipads to Oxford (EGTK) SMR - thanks to @RedstonePilot (Ben Walker)
```

Please be careful to check the effective AIRAC date on the issue to make sure the Changelog entry type and section (and PR milestone) are correct.